contracts-periphery/Makefile

```diff
@@ -10,7 +10,8 @@ endif
 build :; forge build
 clean :; forge clean
 install :; $(INSTALL_CMD)
-test :; forge test
+test :; forge test --nmc DeployBatchSendTest && make test-scripts
 test-gas :; forge test --match-path *.gas.t.sol
+test-scripts :; forge test --mc DeployBatchSendTest --sender 0x4f78F7f3482D9f1790649f9DD18Eec5A1Cc70F86
```

Comment:
Do we need two separate commands here, or can we just use that sender for all tests and change the `test` command to use that sender? Ideally we wouldn't need to fragment things.

If we do fragment things, the `test` command in package.json needs to be updated because it only runs `make test`

Reply:
https://github.com/ScopeLift/umbra-protocol/pull/504/commits/ed081a587b4cf48808b00edf7bb85c4b932608ad
 Using teh same sender for all forge tests is causing a couple `batchSend` tests to fail... gonna look into it and report back

Reply:
Looks like there's now only one test command—what was the issue here?

Reply:
What do you mean? Should I have kept the `test-scripts` one?

Reply:
> [ed081a5](https://github.com/ScopeLift/umbra-protocol/commit/ed081a587b4cf48808b00edf7bb85c4b932608ad) Using the same sender for all forge tests is causing a couple `batchSend` tests to fail... gonna look into it and report back

This was caused by `expectCall` in batchSend tests, so just fixed them up.

Reply:
> Using the same sender for all forge tests is causing a couple batchSend tests to fail... gonna look into it and report back

I was just asking about resolution to this, since you said using the same sender caused the batchSend tests to fail. But sounds like it was really the `expectCall` issue?

Reply:
Yes! Seems that way

Reply:
got it, thanks 👌